pegasus/sites.v3/code.org/public/css/csc-module.scss

```diff
@@ -0,0 +1,59 @@
+$color-black: black;
+$color-white: white;
+
+.csc-module {
+  padding: 1em;
+
+  .csc-module-outer-wrapper {
+    border: 1px solid;
+    border-radius: 8px;
+  }
+
+  a {
+    font-family: 'Gotham 4r', sans-serif;
+    color: $color-black !important;
+    text-decoration: none !important;
+  }
+
+  .csc-module-wrapper {
+    padding: 0 1.5em 1.5em;
+  }
+
+  h3{
```

Comment:
small nit: should there be a space here (same for `a.btn` below)?

Reply:
Ah yes! Good catch 😄